src/linker/Linker.Steps/BodySubstituterStep.cs

```diff
@@ -157,7 +157,7 @@ void ProcessField (TypeDefinition type, XPathNodeIterator iterator)
 
 			var field = type.Fields.FirstOrDefault (f => f.Name == name);
 			if (field == null) {
-				Context.LogMessage (MessageImportance.High, $"Could not find field '{name}' for substitution.");
+				Context.LogMessage (MessageImportance.Low, $"Could not find field '{name}' for substitution.");
```

Comment:
Shouldn't all these cases be errors ?

Reply:
I don't think we have to be pedantic if someone wants to use single xml for multiple purposes

Reply:
The problem is that these can silently stop working etc.

Reply:
Yes but it's not a blocker. Let's go with .Normal